code/modules/events/ion_storm.dm

```diff
@@ -23,7 +23,8 @@
 
 /datum/round_event/ion_storm/announce(fake)
 	if(prob(announceChance) || fake)
-		priority_announce("Ion storm detected near the station. Please check all AI-controlled equipment for errors.", "Anomaly Alert", 'sound/ai/ionstorm.ogg')
+		//priority_announce("Ion storm detected near the station. Please check all AI-controlled equipment for errors.", "Anomaly Alert", 'sound/ai/ionstorm.ogg')
+		priority_announce("Telstra is having an unexpected maintenance on their systems, we apologies for any issues caused.", "Anomaly Alert", 'sound/ai/ionstorm.ogg')

```

Comment:
Grammar and spelling, also modularize this

Reply:
just noticed I screwed up that spelling, oops...